### ISSUE TITLE
您好，关于loss值为负数，这是正常现象吗

### ISSUE THREAD

**Issue by yang88-jun** [Le-Xiaohuai-speech/DPCRN_DNS3#10]

**Comment by Le-Xiaohuai-speech** [comment]
正常啊snr是负数，取了对数之后的mse也可能是对数啊。当然你可以试试各种loss。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年12月28日(周二) 下午3:30
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: [Le-Xiaohuai-speech/DPCRN_DNS3] 您好，关于loss值为负数，这是正常现象吗 (Issue #10)




 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by plutols** [comment]
how much is your loss about your pretrain_model

**Comment by Le-Xiaohuai-speech** [comment]
In fact, I have forgotten...
Reload the weights of the pretrained model and train.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Le-Xiaohuai-speech/DPCRN_DNS3"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年4月2日(星期六) 下午2:59
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Le-Xiaohuai-speech/DPCRN_DNS3] 您好，关于loss值为负数，这是正常现象吗 (Issue #10)





 
how much is your loss about your pretrain_model

**Comment by lacking1** [comment]
How many epochs you trained for pretrained models? Thanks a lot.
